.github/workflows/pr-verifier.yaml

```diff
@@ -0,0 +1,19 @@
+name: PR Verifier
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize]
+
+permissions:
+  checks: write
```

Comment:
Does it really need to write? None of our existing workflows have it.

Reply:
yes, we don't need it, I will remove it.

Reply:
removed